infra/src/main/kotlin/fi/fta/geoviite/infra/split/SplitController.kt

```diff
@@ -1,7 +1,19 @@
 package fi.fta.geoviite.infra.split
 
-import org.springframework.stereotype.Controller
+import fi.fta.geoviite.infra.authorization.AUTH_ALL_WRITE
+import fi.fta.geoviite.infra.common.IntId
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
 
-@Controller
-class SplitController {
+@RestController
+@RequestMapping("/location-track-split")
```

Comment:
Mietin olisiko loogisempaa jos splittaamiselle olisi oma ali-path-avaruutensa `/track-layout/location-tracks/` alla, esim. `/track-layout/location-tracks/split/`

Reply:
Mietin samaa, mutta päädyin tähän ajatuksella että se olisi analoginen linkityksen kanssa (joka on omansa eikä kunkin layout-osion alla). En ole mitenkään täysin vakuuttunut kumminkaan päin. Muilla mielipiteitä?

Reply:
Juusen ehdotus tuntuisi sen suuntaiselta mitä itsekin suosisin. Linkitys voisi kyllä olla myös layout-osion alla (esim. `/track-layout/linking` tai `/link`). 

Tässä tuo Juusen ehdotus `/track-layout/location-tracks/split` tai `/splitting` (-ing-pääte myös linkityksessä) olisi sinänsä looginen, että se liittyy paikannuspohjaan & sijaintiraiteisiin (eikä muihin?). Linkitys kuitenkin liittyy muihinkin elementteihin, jolloin laajempi avaruus `/track-layout/linking` olisi loogisempi. 

Nykyinen pelkkä `/linking` tuntuu siis hieman liian laajalta, sillä eihän se taida olla paikannuspohjan ulkopuolella käytössä?

Reply:
Joo kyllä ne ehdottomasti track-layout:iin liittyy, että siinä mielessä voisi ihan hyvin olla sen allakin. Muistaakseni tuota tehtäessä ajateltiin että linkitys on oma geoviitteen keskeinen termi itsessään ja ansaitsee oman haaransa puussa. Matalammat polut on myös helpompi käyttää/lukea kun termi kuitenkin on yksiselitteinen. Molemmissa toki puolensa,  voisi tuon kyllä siirtäkin layoutin alle.